### DISCUSSION TITLE
Constantly receiving routes-manifest.json error after configuring yarn workspaces

### DISCUSSION THREAD

**Discussion by aceaspades-worldspark** [vercel/vercel#9625] [Help]
Ever since configuring Yarn Workspaces, my teammate and I are _constantly_ receiving the `routes-manifest.json couldn't be found` error. We've tried every single step listed in the help documentation, and a few ad-hoc solutions from the jungles of the internet. 

Sometimes, all it takes is a Redeploy to fix it. Other times, however, are not so easy.

What's _really_ strange is, **the exact same commit can build properly when merged into one branch, but fail deployment when merged onto other branches**. Is that intended behavior?

Does anyone else constantly receive this error?

**EDIT:** Another keen example here. I promoted a successful deployment to Production. It built wonderfully. I changed a shared environment variable, then re-deployed the Production build, using the existing build cache. Bam, same error, routes-manifest.json couldn't be found.

**Comment by amunrarara** [comment]
@aceaspades-worldspark Following from my main account
